src/app/main/directive/invalid-control-scroll.directive.spec.ts

```diff
@@ -37,7 +37,7 @@ class TestLinkHostComponent implements OnInit {
     }
 
     onSubmit() {
-        console.log('form submitted');
+        console.warn('form submitted');
```

Comment:
Does it make difference have it as a `warn` instead of `log`?

Reply:
This is the question: https://docs.google.com/document/d/1SIVMxoV0N8xYqhVk_0hhuqSuUF47D1YYrDESqdmipfM/edit?disco=AAAALqCScG4

Reply:
I see, but it doesn't answer my question ;)
Why you exclude error and log but keep the others @flavens ?

Different console options can have different usages. In the example above we get warn the code is submitted? It should be rather info or log :)

All console methods can be divided in some groups. The group with `log info warn error` is to display messages. The only one difference here are is visual, which should help to identify errors quicker. The common practice is to use `log`. And I think it's fine, unless there are some sophisticated rules to use all others (which is always nice to have ;).

Anyways, I find this linting rule just a matter of preferences, because at the end, no console logs, warns, errors, should be on the production (there is a flag to disable these). But strange would be to get most commonly used 'log' and 'error' w/o having the rules set for the others. Just my 2 Rappens ;)

Reply:
@mpro7 Thanks for the feedback. Would you like to copy this comment to the DSP-UI design documentation on google drive? Because we agreed last Friday to have the discussion there. 
About "no console.log, console.warn and console.error": it was just an idea. And as you said we all have to agree to the conventions. I can revert the rule. No problem. But there was no other feedback in the design documentation than the one from @flavens.

Reply:
Okay.... I misunderstood the comment from @flavens 
I will fix it. I'm sorry 🙈

Reply:
Ok, I will.

Reply:
This is now the "correct" (has to be discussed) solution for `console.log, .error and .warn`: 4f3d7cc and fb7ded2
I'm sorry again!

Reply:
> Ok, I will but this is why I wanted to have rather some ideas rose on the chat, where we can get notification. I' busy with other things and don't have much time to go through all rules, but I can always add my 2 Rappens :D

Yes, I updated the design documentation, added my comment and then I opened the new thread in the R&D chat room. But nobody answered in the chat; only Flavie added a comment to the design documentation. So, how we can we all agree to the rules (now)?

Reply:
if all thre rules have been defined in the eslint file in this PR, each approval = agreement

Reply:
@kilchenmann did you go through all the rules and if, did you find anything which would overwhelming?

Reply:
I went through most of the options and looked for inconsistencies. In this PR `eslintrc.js` was created from `tslint.json` where we had already most of the rules configured. ESLint is just more restrict and this is why we have so many changes in this PR.

Reply:
That's fine. Have a look on [my newest comment](https://github.com/dasch-swiss/dsp-app/pull/394#discussion_r585524067).